src/StreamDeckLib/ConnectionManager.cs

```diff
@@ -170,7 +170,7 @@ private async Task Run(CancellationToken token)
 
 				var jsonString = await _proxy.GetMessageAsString(token);
 
-				if (!string.IsNullOrEmpty(jsonString) && !jsonString.StartsWith("\0"))
+				if (!string.IsNullOrEmpty(jsonString) && !jsonString.StartsWith("\u0000", StringComparison.OrdinalIgnoreCase))
```

Comment:
Note - if this lib is updated to .NET Standard 2.1, `.StartsWith('\0')` would work here as well. Decided to go with this change to prevent needing dependents to update.

https://docs.microsoft.com/en-us/dotnet/api/system.string.startswith?view=netcore-3.1#System_String_StartsWith_System_Char_